e2e-test/teaspoons_gcp_e2e_test.py

```diff
@@ -1,232 +1,14 @@
 from workspace_helper import create_gcp_workspace, delete_workspace, share_workspace_grant_owner, add_wdl_to_gcp_workspace
```

Comment:
should we rename this file to `teaspoons_gcp_service_e2e_test_setup.py` to keep it consistent with the cli name?

Reply:
yeah probably without the _setup suffix since this one does the test itself and not just setup